test/actions.test.js

```diff
@@ -531,7 +268,7 @@ test('[actions.diff] success', function(assert) {
     SecretPassword: 'secret'
   };
 
-  actions.diff('my-stack', 'us-east-1', url, parameters, function(err, data) {
+  actions.diff('my-stack', 'us-east-1', 'UPDATE', url, parameters, function(err, data) {
```

Comment:
I was looking for some test where `changeSetType` was set to `CREATE` but I didn't see any. Did I overlook? Or is that not really a testing gap?